### DISCUSSION TITLE
EmulatorJS Storage Question(s)

### DISCUSSION THREAD

**Discussion by Cattn** [EmulatorJS/EmulatorJS#469] [Q&A]
So as far as I'm aware, there's no way to download/export all your saves at once. I know they are saved in indexeddb (I'm not that knowledged on how that works), but I'm in the process of creating an option for users to export all of their site data, and am trying to include game saves, is there anyway I could get all of the saves and serve them as a download? (preferably in some sort of usable state)

**Comment by Cattn** [comment]
I was thinking something along the lines of users being able to create acc/login with google and have saves for lighter consoles (gb/gba/nes) be saved for them to use anywhere

**Reply by Ramaerel** [reply]
Hey, if you check out issue number 235, there's code in there that you can use to grab SRAM saves from indexeddb. I can't access my computer for a few weeks but if you still need help then, one of my projects has a working example

**Reply by Cattn** [reply]
I see, thank you! Will get back once I try it out

**Reply by Cattn** [reply]
hey @Ramaerel Could you show me an example when you have the chance?
